puppeteerScript.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * MIT License
+ * Copyright (c) 2021 Red Hat One Platform
+ *
+ * Puppeteer Script to bypass the login page with lighthouse.
+ *
+ * @author Rigin Oommen <riginoommen@gmail.com>
+ *
+ * @requires SSO_USERNAME Managed in the jenkins credentials store.
+ * @requires SSO_PASSWORD Managed in the jenkins credentials store.
+ *
+ * Created at     : 2021-07-09 16:18:34
+ * Last modified  : 2021-07-09 16:23:59
+ */
+
+
+module.exports = async ( browser ) => {
+  console.info('Puppeteer script execution started');
+  const page = await browser.newPage();
+  const navigationPromise = page.waitForNavigation();
+  await page.goto('https://one.redhat.com', {
```

Comment:
Can you use an environment variable for this too? We should not hard code any urls meant for internal purpose.

Reply:
Changed to URLs to env variables